website/source/docs/commands/validate.html.md.erb

```diff
@@ -28,6 +28,14 @@ and supports `go-getter` syntax.
 On successful validation, exit code 0 will be returned, otherwise an exit code
 of 1 indicates an error.
 
+Service names are validated according to [RFC-1123](https://tools.ietf.org/html/rfc1123).
```

Comment:
I would remove from here and instead add to: https://www.nomadproject.io/docs/job-specification/service.html#name.

Reply:
I am not a fan of any specific validation details on this page since then we will end up documenting validation details here and on the specific stanzas.